packages/common/src/utils/linking.ts

```diff
@@ -0,0 +1,18 @@
+export const externalLinkAllowList = new Set([
+  'facebook.com',
+  'instagram.com',
+  'tiktok.com',
+  'twitter.com',
+  'x.com',
+  'blog.audius.co'
+])
+
+export const isAllowedExternalLink = (link: string) => {
+  try {
+    let hostname = new URL(link).hostname
```

Comment:
Does react-native support `URL.hostname` now? Nice!

Reply:
yeah it does thank heavens